Specs/Widgets/Viewer/ViewerSpec.js

```diff
@@ -1021,4 +1025,165 @@ defineSuite([
         expect(preMixinDataSource.entities.collectionChanged._listeners.length).not.toEqual(preMixinListenerCount);
         expect(postMixinDataSource.entities.collectionChanged._listeners.length).not.toEqual(postMixinListenerCount);
     });
+
+    it('zoomTo throws if target is not defined', function() {
+        viewer = createViewer(container);
+
+        expect(function() {
+            viewer.zoomTo();
+        }).toThrowDeveloperError();
+    });
+
+    it('flyTo throws if target is not defined', function() {
+        viewer = createViewer(container);
+
+        expect(function() {
+            viewer.flyTo();
+        }).toThrowDeveloperError();
+    });
+
+    it('zoomTo when target is Cesium3DTileset and uses default offset when offset not defined', function() {
+        viewer = createViewer(container);
+         var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+         var tileset = new Cesium3DTileSet({
+            url : path
+         });
+
+        // stored for movement check
+        var camPos = Cartesian3.clone(viewer.camera.position);
+        var camDir = Cartesian3.clone(viewer.camera.direction);
+
+        //spyOn(viewer.camera, 'viewBoundingSphere');
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+
+            //var boundingSphere = tileset.boundingSphere;
+            //var offset = new HeadingPitchRange(0.0, 0.0, 2.0 * boundingSphere.radius);
+            return viewer.zoomTo(tileset).then(function() {
+                //expect(viewer.camera.viewBoundingSphere).toHaveBeenCalledWith(boundingSphere, offset);
+
+                expect(viewer.camera.position).not.toEqual(camPos);
+                expect(viewer.camera.direction).not.toEqual(camDir);
+            });
+        });
+    });
+
+    it('flys to target when target is Cesium3DTileset and uses default offset when options is not defined', function() {
+        viewer = createViewer(container);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileSet({
+            url : path
+        });
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+            var promise = viewer.flyTo(tileset);
+            var wasCompleted = false;
+
+            spyOn(viewer.camera, 'flyToBoundingSphere').and.callFake(function(target, options) {
+                wasCompleted = true;
+                options.complete();
+            });
+
+            viewer.render();
+
+            return promise.then(function() {
+                expect(wasCompleted).toEqual(true);
+            });
+        });
+
+    });
+
+    it('flys to when target is Cesium3DTileset and uses default offset when offset not defined in options', function() {
```

Comment:
Just to make sure we're not breaking functionality, it might be best to add some tests to check zooming to entities. You should be able to duplicate this test and just pass an entity or array of entities instead of a tileset.